### ISSUE TITLE
[BUG] PAKS Motorola Admin app

### ISSUE THREAD

**Issue by RTB713** [samolego/Canta#25]
**Describe the bug**
A clear and concise description of what the bug is. Application called PAKS and PAK notifications on Motorola phones. I believe it has something to do with installing 3rd party apps. This app acts as an Admin. Sometimes I try to do things in my phone and am "blocked by IT" OR "Blocked by Admin" I use canta to uninstall, go back 3 mins later and it's right back. Please figure a way to keep this deleted. To be told I can't do something on my phone because I'm block by so and so is INFURIATING!!! Thank you.
![Screenshot_20240117-165157](https://github.com/samolego/Canta/assets/156967931/710cb671-acd2-4853-8e42-96d1c7487a42)

**To Reproduce**

**Expected behavior**

**Version info**
 - Android version: 
 - Canta version:
 - Shizuku version:

**Logcat**

<details>
  <summary>View logcat</summary>
  ```logcat
  paste logcat here
  ```
</details>

**Comment by samolego** [comment]
Closing as a duplicate of #24
